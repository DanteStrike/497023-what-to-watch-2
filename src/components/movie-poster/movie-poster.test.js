import React from "react";
import renderer from "react-test-renderer";
import MoviePoster from "./movie-poster.jsx";


it(`Render correctly MoviePoster component`, () => {
  const component = renderer
    .create(
        <MoviePoster
          isBig={false}
          image={`url`}
          name={`any`}
        />
    ).toJSON();

  expect(component).toMatchSnapshot();
});
