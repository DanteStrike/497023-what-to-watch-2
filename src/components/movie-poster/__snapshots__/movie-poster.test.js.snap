// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render correctly MoviePoster component 1`] = `
<div
  className="movie-card__poster"
>
  <img
    alt="any"
    height="327"
    src="url"
    width="218"
  />
</div>
`;
